cdhweb/resources/models.py

```diff
@@ -81,3 +82,22 @@ class LandingPage(Page, RichText):
         upload_to=upload_to("resources.landing_pages.image", "resources"),
         format="Image", max_length=255, null=True, blank=True)
 
+
+class PublishedQuerySetMixin(models.QuerySet):
+    '''QuerySet mixin. Adds a filter to find published content.
+    Uses the published check from
+     :class:`mezzanine.core.managers.PublishedManager`.'''
+
+    def published(self, for_user=None):
+        """
+        For non-staff users, return items with a published status and
+        whose publish and expiry dates fall before and after the
+        current date when specified.
+        """
+        if for_user is not None and for_user.is_staff:
```

Comment:
codecov thinks we aren't testing the first case (return all). not sure if that matters.